### PR TITLE
search through source manifests to determine operator disabled

### PR DESCRIPTION
**Pull request by joelddiaz**
When the render process is run during bootstrap, it will check all files in the destination directory (which also contains the install-time manifests) for the CCO ConfigMap that enables/disables the operator.

This should allow an install process where a user can provide their own install-time manifests with the CCO ConfigMap defined to keep CCO from ever running in the cluster. The user-provided ConfigMap will make it into the cluster during the bootstrap stage, so when the in-cluster CCO is installed, it will see the ConfigMap and never attempt to process CredentialsRequests.

### CODE REVIEW

**Comment by joelddiaz** [comment]
@dgoodwin by making this new parameter optional, it means we need changes into the installer (simple changes).

In reality, the --dest-dir location mounted into the pod is where the manifests already are so we could accomplish this with the existing mountpoints loaded into the pod. But in the future if the installer is refactored so that this isn't the case, having the explicit parameter will make it more clear that additional consideration is needed. WDYT?

File: cmd/manager/render.go [hunk 1]
```diff
@@ -115,22 +124,76 @@ func runRenderCmd(cmd *cobra.Command, args []string) {
...
+		decoder := yaml.NewYAMLOrJSONDecoder(file, 4096)
+		configMap := &corev1.ConfigMap{}
+		if err := decoder.Decode(configMap); err != nil {
+			log.WithError(err).Debug("failed to decode into configmap")
+			continue
+		}
+
+		if configMap.Namespace == minterv1.CloudCredOperatorNamespace && configMap.Name == minterv1.CloudCredOperatorConfigMap {
+
+			disabled, err := utils.CCODisabledCheck(configMap, &log.Logger{Level: log.GetLevel()})
```
Comment (staebler) [inline cmd/manager/render.go]: This does not set up the logger sufficiently and will cause a panic if the logger is ever used. Use `log.New` instead.

**Comment by dgoodwin** [comment]
So the manifest render command is called by the installer, and --dest-dir is where we're supposed to drop our manifests, but if the user injected custom manifests, these are also present there?

If so I think that's enough API stability, I can't envision they would start telling us to write somewhere manifests would not exist anymore. 

It would be simpler to avoid the installer change, I'd say lets assume dest-dir points to the manifests dir.

This is a nice clever solution to the problem.

**Comment by joelddiaz** [comment]
@dgoodwin ok, removed the --manifests-dir parameter and will just search through the destination dir for the CCO configmap. No installer changes needed with this approach.

**Pull request merged**